flex/validation/common.py

```diff
@@ -436,10 +437,13 @@ def generate_value_processor(type_, collectionFormat=None, items=None, **kwargs)
         assert False, "This should not be possible"
     else:
         if type_ == ARRAY and collectionFormat:
-            delimeter = DELIMETERS[collectionFormat]
-            # split the string based on the delimeter specified by the
-            # `collectionFormat`
-            processors.append(operator.methodcaller('split', delimeter))
+            if collectionFormat in DELIMETERS:
+                delimeter = DELIMETERS[collectionFormat]
+                # split the string based on the delimeter specified by the
+                # `collectionFormat`
+                processors.append(operator.methodcaller('split', delimeter))
+            else:
+                assert collectionFormat == MULTI, "Unsupported collectionFormat"
```

Comment:
Done. I've also changed the wording to make it clear that the error might not be in the file to be checked.